feature/src/main/java/com/teamkkumul/feature/utils/time/timeUtil.kt

```diff
@@ -25,4 +25,20 @@ object TimeUtils {
         val date = inputFormat.parse(this)
         return outputDateFormatText.format(date)
     }
+
+    fun String.calculateDday(): Pair<String, Int> {
+        val eventDate = outputDateFormat.parse(outputDateFormat.format(inputFormat.parse(this)))
+        val currentDate = outputDateFormat.parse(outputDateFormat.format(System.currentTimeMillis()))
+
+        val diffInMillis = eventDate.time - currentDate.time
+        val dDay = -(diffInMillis / (1000 * 60 * 60 * 24)).toInt()
+
+        val dDayString = when {
+            dDay > 0 -> "D+$dDay"
+            dDay == 0 -> "D-DAY"
+            else -> "D$dDay"
+        }
```

Comment:
p1 : 만들어 놓은 util함수 사용해주세요! 반복되는 코드는 보일러 플레이트를 유발합니다
```suggestion
        val dDayString =setDday(dDay)
```